recipes/ad3/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ad3-cpp" %}
+{% set version = "2.5.4" %}
+#{% set sha256 = "b2ada6f86faf5c9032e8b1a7b8cc0dff68a229e90ad80b366f5564fec03caa17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #path: ../
+  #fn: ad3-{{ version }}.tar.gz
+  url: https://github.com/Sebastianborszcz/AD3
+  #sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - eigen
+
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/ad3           # [unix]
+    - test -f ${PREFIX}/include/ad3/Factor.h  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\ad3\Factor.h (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/DerThorsten/AD3
+  license: GPL3
+  license_family: GPL3
+  license_file: LICENSE
+  summary: 'AD3 Alternating Directions Dual Decomposition'
+  description: 'AD3 Alternating Directions Dual Decomposition'
+  documentation_url:  https://github.com/DerThorsten/AD3
+  dev_url: https://github.com/DerThorsten/AD3
+
+extra:
+  recipe-maintainers:
+- DerThorsten
```

Comment:
The recipe maintainer needs to be indented. Also, you should add yourself as a recipe maintainer and ping the author if you think they might be interested in co-maintaining the recipe with you.